src/Speckle.Objects/Interfaces.cs

```diff
@@ -118,14 +106,41 @@ public interface IDisplayValue<out T> : ISpeckleObject
   T displayValue { get; }
 }
 
-public interface IDataObject : ISpeckleObject
+#endregion
+
+#region GIS
+public interface IGisFeature : ISpeckleObject
 {
-  string name { get; }
+  Base attributes { get; set; }
+}
+
+#endregion
+
+#region Data objects
 
-  [DetachProperty]
-  IReadOnlyList<Base> displayValue { get; }
+/// <summary>
+/// Specifies properties on objects to be used for data-based workflows
+/// </summary>
+public interface IProperties : ISpeckleObject
+{
+  Dictionary<string, object?> properties { get; }
+}
+
+public interface IDataObject : ISpeckleObject, IProperties
```

Comment:
Did we want `IDataObject : IDisplayValue`?

Is there any value still in `IDisplayValue`? or shall we kill it

Reply:
Ah, I see your comment (tho you could have made it implement `IDisplayValue<IReadOnlyList<Base>?>`)

But I'd like to discuss both the nullability, and existence of `IDisplayValue`

Reply:
Agreed with Claire `IDataObject : IDisplayValue` with the intention of maybe cleaning up and removing this interface in future